firmware/shared/controls/motor_torque.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+#include <tuple>
+
+#include "shared/util/moving_average.h"
+
+// Peter Jabra and Aleeza Ali Zar
+
+enum class State {
+    Run,
+    Stop,
+};
+
+namespace ctrl {
+
+template <typename T>
+std::tuple<T, T> CalculateMotorTorque(T new_torque_value, T right_factor,
+                                      T left_factor, bool reset = false) {
+    static shared::util::MovingAverage<T, 10> running_average;
+
+    if (reset) {
+        running_average = shared::util::MovingAverage<T, 10>();
+    }
+
+    running_average.LoadValue(new_torque_value);
+
+    T running_average_value = running_average.GetValue();
+    T scaled_running_average = running_average_value * 10;
```

Comment:
what is this `*10` for?

Reply:
So the numbers passed into this function will be inherited from previous functions which return a number from 1-100, however, we need a output a number from 1- 1000 as thats what is used the later functions. While we could change the scale to just be smaller, im not sure how sensitive later functions will be. They might require numbers between 1-1000, so for now i believe we should keep it as is, and leave a note to scale it down if we find out later functions dont require sensitive values

Reply:
let's remove it from this function. it sound like the scale factor is more of an interface between the functions, so we can do the math there. i think it muddles the behaviour of this one